benchmarks/examples/amd_openmpi_sample_sbatch_script

```diff
@@ -2,21 +2,21 @@
 #SBATCH --job-name=testMPI
 #SBATCH --output=output.txt
 #SBATCH --partition=normal
-#SBATCH -N 2
+#SBATCH -N 2 --nodelist=springnode00003.spring.test,springnode00004.spring.test
```

Comment:
since its a example let the node names be amdnode1.spring.test,amdnode2.spring.test